README.md

```diff
@@ -25,6 +25,43 @@
   open index.html
 ```
 
+## For Visual Studio Code users 
+Here is the `launch.json` configuration for running the server and the client simultaneously
```

Comment:
I am putting this note about the `launch.json` configuration in the readme instead of the repository because not everyone uses VSCode, and we don't want to push up our `.vscode` directory to the repo.